### PR TITLE
Add parameters file to interthreshold calib example

### PR DESCRIPTION
**Pull request by wictus**
At last documentation is done. Userparams file is also submitted to the sphinx.

### CODE REVIEW

File: InterThresholdCalibration/PARAMETERS.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# InterThresholdCalibration parameters description
+
+## Contents
+Description of available parameters in InterThresholdCalibration example. Note that configuration/calibration files can be obtained from [PetWiki](http://koza.if.uj.edu.pl/petwiki/index.php/Default_settings_and_parameters_used_in_the_analyses)
+
+- `Save_Cotrol_Histograms_bool`
```
Comment (alekgajos) [inline InterThresholdCalibration/PARAMETERS.md]: In the meantime we fixed the typo, it's now `Save_Control_Histograms_bool`

File: InterThresholdCalibration/PARAMETERS.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- `InterThresholdCalibration_TimeCalibRunNumber_int`  
+Number of calibration run associated with Acquisition Campaign. Default value: `4`
+
+- `InterThresholdCalibration_Frac_err_double`  
+maximal fractional uncertainty of parameters accepted by calibration. Default value: `0.3`
+
+- `InterThresholdCalibration_Min_ev_int`  
+minimal number of events for a distribution to be fitted. Default value: `100`
+
+- `InterThresholdCalibration_TimeConstantsInterThrCalibOutputFile_std`
```
Comment (alekgajos) [inline InterThresholdCalibration/PARAMETERS.md]: Shouldn't it end with "_std::string"?

File: InterThresholdCalibration/PARAMETERS.md (hunk 1 above)
Comment (wictus) [reply InterThresholdCalibration/PARAMETERS.md]: Done

File: InterThresholdCalibration/PARAMETERS.md (hunk 2 above)
Comment (wictus) [reply InterThresholdCalibration/PARAMETERS.md]: Done

**Comment by wictus** [comment]
I removed part of description which is already in other example as suggested.

**Pull request merged**